src/components/Settings/Organization/AccountLists/AccountListRow/AccountListRow.test.tsx

```diff
@@ -97,42 +124,103 @@ describe('AccountLists', () => {
   });
 
   describe('Handling Deletions', () => {
```

Comment:
Can you ensure the user, accountLkist, organization are gone from the page after a successfull mutation?